cli/tools/lint.rs

```diff
@@ -277,24 +278,27 @@ pub fn format_diagnostic(
 ) -> String {
   let mut lines = vec![];
 
-  for i in range.start.line..=range.end.line {
-    lines.push(source_lines[i - 1].to_string());
```

Comment:
The transition away from 1-indexed non-display values hasn't completed yet, but the change was done in deno_lint (except for json serialization).